__tests__/app.test.js

```diff
@@ -31,22 +32,61 @@ describe("app", () => {
             expect(result.body.topics.length).toBe(3);
             result.body.topics.forEach((topic) => {
               expect(typeof topic.description).toBe("string");
-                expect(typeof topic.slug).toBe("string");
-                
+              expect(typeof topic.slug).toBe("string");
             });
-              console.log(result)
+            console.log(result);
           });
       });
     });
-  })
-    describe("err handeling ", () => {
-        test("when and url is invalid, respond with a 404", () => {
-            return request(app)
-                .get("/api/setirsti")
-                .expect(404)
-                .then(({ body }) => {
-                    expect(body.msg).toBe("please enter a valid url");
-                })
-        })
-    })
+    describe("/api/articles/:article_id", () => {
+      test("GET: 200, respond with the article corosponding to the article id", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then((result) => {
+            expect(typeof result.body.article.author).toBe("string");
+            expect(typeof result.body.article.title).toBe("string");
+            expect(typeof result.body.article.article_id).toBe("number");
+            expect(typeof result.body.article.body).toBe("string");
```

Comment:
As we are expecting a specific article It is important we check the article_id specifically here to ensure we get article 1